modules/gdscript/gdscript_byte_codegen.cpp

```diff
@@ -529,6 +529,7 @@ void GDScriptByteCodeGenerator::write_construct(const Address &p_target, Variant
 		append(p_arguments[i]);
 	}
 	append(p_target);
+	alloc_call(p_arguments.size());
```

Comment:
Would there be the same issue with `write_construct_array` and `write_construct_dictionary`?

Reply:
@akien-mga `Array` and `Dictionary` are constructed locally but other types are constructed by calling the constructor(`Variant::construct()`) at ` GDScriptFunction::call()`  which seems inconsistent?